packages/snap-preact/components/src/themes/bocachica/components/molecules/grid.ts

```diff
@@ -0,0 +1,4 @@
+import type { GridProps } from '../../../../components/Molecules/Grid';
+
+// Grid component props
+export const grid: Partial<GridProps> = {};
```

Comment:
should this have the stylescript prop even though its empty like the other components?

Reply:
This will be added once the style is completed.